config/middleware.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const morgan = require('morgan');
+const helmet = require('helmet');
+const cors = require('cors');
+
+// MIDDLEWARE
+module.exports = server => {
+  server.use(helmet());
+  server.use(morgan('dev'));
+  server.use(express.json());
+  server.use(cors());
+};
```

Comment:
Great job breaking this into it's own file and using all of your middleware here.

Reply:
I also can't see where you're using your routes. You would want to use them here. I might be seeing an older push though.